controller.rst

```diff
@@ -666,4 +734,5 @@ Learn more about Controllers
 
     controller/*
 
+.. _`helper methods`: https://github.com/symfony/symfony/blob/master/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php
```

Comment:
I'm not sure why, but the API docs for the trait are *terrible*, basically empty :/. It's actually a problem because I *would* like to link to some methods in the trait via `:method` in a few places - http://api.symfony.com/master/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.html